corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -89,14 +95,44 @@ def _copy_unprocessed_forms(self):
             _migrate_form_attachments(sql_form, couch_form)
             _migrate_form_operations(sql_form, couch_form)
 
-            diffs = json_diff(couch_form.to_json(), sql_form.to_json(), track_list_indices=False)
-            self.diff_db.add_diffs(
-                'form', couch_form.form_id,
-                _filter_form_diffs(couch_form.doc_type, diffs)
-            )
+            if couch_form.doc_type != 'SubmissionErrorLog':
+                diffs = json_diff(couch_form.to_json(), sql_form.to_json(), track_list_indices=False)
+                self.diff_db.add_diffs(
+                    'form', couch_form.form_id,
+                    _filter_form_diffs(couch_form.doc_type, diffs)
+                )
 
             _save_migrated_models(sql_form)
 
+    def _copy_unprocessed_cases(self):
```

Comment:
Not too concerned about the correctness of this since if the cases get undeleted they will also get re-built from the forms.